### PR TITLE
 Fix - 1752 - Adding Video For Installation of Badgeyay Front-End in Local Environment

### LINKED ISSUE #1752: Adding Video For Installation of Badgeyay  Front-End in Local Environment
**I'm submitting a ...**
- [x] feature request

No video showing installation for Front-End

### PR DESCRIPTION
**Pull request by sk9331657**
Fixes #1752 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

-Added Video fro frontend

### CODE REVIEW

File: frontend/README.md [hunk 1]
```diff
@@ -19,6 +19,10 @@ You will need the following things properly installed on your computer.
 
 ## Installation
 
+##Video Installation
```
Comment (ParthS007) [inline frontend/README.md]: Correct the Markdown and use `### Video Installation`.

Comment (sk9331657) [reply frontend/README.md]: Fixed

**Comment by sk9331657** [comment]
@ParthS007  This can be merged.

**Comment by sk9331657** [comment]
@ParthS007  I have made changes. Please fix this also.

**Comment by sk9331657** [comment]
@yashLadha  Event thought instructions are present still developers are facing problem.

**Comment by yashLadha** [comment]
Maybe they are doing it the wrong way and instructions are crystal clear. I don't know which instruction you are referring too?

**Comment by sk9331657** [comment]
@yashLadha  According to me it will be helpful, rest is up to you. :+1:

**Comment by ParthS007** [comment]
@sk9331657 Video Installation is good for backend but Frontend have very easy steps to install so It is not required as of now as said by @yashLadha.
Closing this :

**Pull request closed without merging**